packages/containers/sandbox/src/app/expressions/index.js

```diff
@@ -0,0 +1,9 @@
+import { selectors } from '@talend/react-cmf';
+
+function getUsersCollectionAsArray({ context }, collectionPath) {
+	const list = selectors.collections.get(context.store.getState(), collectionPath);
+
+	return list && list.toJS() ? list.toJS() : [];
```

Comment:
FYI: this create a new reference on each store mutation.
It would be nice if memoize it